packages/toast/src/FzToastQueue.vue

```diff
@@ -37,16 +42,29 @@
 </template>
 
 <script setup lang="ts">
-import { computed, ref, toRef } from "vue";
+import { computed, ref, watch } from "vue";
 import { FzToast, FzToastQueueProps, Toast } from "./index";
 import { toasts as internalToasts, removeToast } from "./queue";
 
 const props = withDefaults(defineProps<FzToastQueueProps>(), {
   align: "right",
+  openOnNewToast: true,
 });
-const toasts = props.toasts ? toRef(props.toasts) : internalToasts;
+
+const toasts = computed(() => props.toasts ?? internalToasts.value);
 const toastsHeight = ref<number[]>([]);
 const isHovering = ref(false);
+const toastsLength = computed(() => toasts.value.length);
+let currentTimeoutId: number | undefined = undefined;
+
+watch(toastsLength, (newValue, oldValue) => {
+  if (!props.openOnNewToast) return;
+  if (newValue <= oldValue) return;
+  if (currentTimeoutId != null) window.clearTimeout(currentTimeoutId);
+  
+  isHovering.value = true;
+  currentTimeoutId = window.setTimeout(() => (isHovering.value = false), 2000);
```

Comment:
I suggest adding the ability to customize the maximum timeout duration, perhaps via a custom props ( with the default value of 2000)

Reply:
fair enough